### DISCUSSION TITLE
Связь между полями

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#100] [Как это сделать?] [answered]
Можно ли создать связь между полями?
Пример:
Поле Задача есть в "Цикловой таблице" и есть в "Расчётной в цикле".
Нужно чтоб изменение текста которое произошло в "Цикловой таблице" произошло в поле которое содержится в "Расчётной в цикле", так же и обратно. Можно ли такое осуществить?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
https://ru.docs.totum.online/label-type-fields — посмотрите целиком, там во второй половина такой пример.
